examples/react-native/package.json

```diff
@@ -1,37 +1,44 @@
 {
   "name": "example-react-native",
   "version": "1.0.0",
-  "main": "./node_modules/react-native-scripts/build/bin/crna-entry.js",
   "private": true,
-  "license": "MIT",
+  "main": "node_modules/expo/AppEntry.js",
   "scripts": {
-    "start": "react-native-scripts start",
     "build": "echo \"Error: no build specified\" && exit 0",
-    "eject": "react-native-scripts eject",
-    "android": "react-native-scripts android",
-    "ios": "react-native-scripts ios",
+    "start": "expo start",
+    "android": "expo start --android",
+    "ios": "expo start --ios",
+    "eject": "expo eject",
     "test": "jest"
   },
-  "devDependencies": {
-    "babel-preset-react-native": "4.0.1",
-    "jest": "21.2.1",
-    "jest-expo": "24.0.0",
-    "react-native-scripts": "1.14.1",
-    "react-test-renderer": "16.8.4"
-  },
   "dependencies": {
     "@ptomasroos/react-native-multi-slider": "1.0.0",
-    "expo": "25.1.2",
+    "algoliasearch": "3.32.1",
+    "expo": "32.0.6",
     "lodash": "4.17.11",
-    "prop-types": "15.6.0",
-    "react": "16.8.4",
-    "react-dom": "16.8.4",
-    "react-instantsearch": "5.4.0",
-    "react-native": "0.52.0",
+    "prop-types": "15.7.2",
+    "react": "16.5.0",
+    "react-instantsearch-core": "5.4.0",
+    "react-instantsearch-native": "5.4.0",
+    "react-native": "https://github.com/expo/react-native/archive/sdk-32.0.0.tar.gz",
```

Comment:
I wonder if renovate will be able to update this

Reply:
That's part of CISA. We don't want Renovate to update this because the Expo SDK is not compatible with all the React Native versions.